core/domain/activity_jobs_one_off_test.py

```diff
@@ -93,3 +93,7 @@ def mock_add_documents_to_index(docs, index):
             self.assertIn('%s' % index, ids)
             self.assertIn('title %d' % index, titles)
             self.assertIn('category%d' % index, categories)
+
+        self.assertIsNone(
```

Comment:
What does this really test?

Reply:
This tests the pass statement in `activity_jobs_one_off.IndexAllActivitiesJobManager.reduce()` function returns None